src/getHumanAge.test.js

```diff
@@ -7,4 +7,54 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should be return current array [0, 0]', () => {
```

Comment:
Please change all your summaries. It should be understandable from the summary what is checked by every check. For now, it is not clear at all.